### PR TITLE
Windoor Recode + Req Windoors

### PR DESCRIPTION
**Pull request by Hathkar**
## About The Pull Request

Changed the parent type of brigdoor to secure, made brigdoors are child of secure doors.
Changed the windoor assembly to use secure instead of brigdoors.
Reorganized the windoors by type.

## Why It's Good For The Game

Basically, windoor code was a big jumbled mess, with brigdoor being defined but only used for half the windoors defined.

## Changelog
:cl:
add: Req windoors are now secure windoors
code: Made secure windoors a parent type
code: Rearranged var-edits on the map files to be in alphabetical order
/:cl:

### CODE REVIEW

**Comment by Hathkar** [comment]
Once the other map PRs are dealt with, I'll update them for this PR.

File: _maps/map_files/MarineHQ/MarineHQ.dmm [hunk 1]
```diff
@@ -1253,8 +1253,8 @@
 "eu" = (
 /obj/structure/sink{
 	dir = 8;
-	pixel_z = 1;
-	pixel_w = -9
+	pixel_w = -9;
```
Comment (spookydonut) [inline _maps/map_files/MarineHQ/MarineHQ.dmm:1256]: hello strongdmm

File: _maps/map_files/MarineHQ/MarineHQ.dmm [hunk 2]
```diff
@@ -2671,8 +2671,8 @@
 /area/mainship/living/grunt_rnr)
 "iU" = (
 /obj/machinery/door/airlock/mainship/generic{
-	name = "\improper Extended Operations Bunks";
-	dir = 8
+	dir = 8;
```
Comment (spookydonut) [inline _maps/map_files/MarineHQ/MarineHQ.dmm:2674]: etc

**Review by spookydonut — COMMENTED** [review]
strongdmm is bad at mapmerging

**Comment by Hathkar** [comment]
> 
> 
> strongdmm is bad at mapmerging

Weird, the hook I have for gitkraken should be taking care of the mapmerges.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**